app/connectors/schema_mapping.py

```
@@ -0,0 +1,59 @@
+        self, row: dict[str, Any], operation: str
+    ) -> str | None:
+        """Evaluate rules in priority order; return first match."""
+        context = {"row": row, "operation": operation, **row}
+        for rule in sorted(self.event_type_rules, key=lambda r: -r.priority):
+            if rule.condition is None:
+                return rule.event_type
+            try:
+                if eval(rule.condition, {"__builtins__": {}}, context):
+                    return rule.event_type
+            except Exception:
+                continue
```

Ruff (0.15.5)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)